### ISSUE TITLE
NEI searching problem

### ISSUE THREAD

**Issue by lavenki** [rwtema/extrautilities_old_issues#273]
Using search in NEI for other mods like so,
![2014-08-24_19 58 12](https://cloud.githubusercontent.com/assets/8536809/4023355/2f2acc76-2b89-11e4-9ccc-6a31f4dfc9af.png)
Not a problem at all, but when i look up Extra Utilities, i get this
![2014-08-24_19 58 41](https://cloud.githubusercontent.com/assets/8536809/4023357/5899c120-2b89-11e4-856a-bdfe09954bf7.png)
Everything disappears even the tool tips while highlighting. But searching a specific item from the Mod such as the Watering Can doesn't affect NEI at all.
![2014-08-24_20 00 11](https://cloud.githubusercontent.com/assets/8536809/4023361/920a3e30-2b89-11e4-9ecc-78191773f4fb.png)

**Comment by rwtema** [comment]
Check the readme. I don't support 1.6.4.

**Comment by lavenki** [comment]
Well atleast tell me what the hell is going on there. If you don't support it anymore then just tell me what to do because it broke my world after i found a sigil in a mineshaft chest.

**Comment by 61352151511** [comment]
By not supporting he means he won't fix bugs for that version, there may be no fix to your problem ever. What you should do is the same as what everyone else playing 1.6.4 should do, update to 1.7.10, if you are waiting on other mods updating just install them later, A lot of mods are actually updated to 1.7.10 now, I've got at least 90 in my modpack. So either deal with a broken world or update.

**Comment by lavenki** [comment]
I know that he's not going to provide a fix, my last EU install was way back 1.5.2, can i just ask what 1.6 version doesn't have this kind of bug? Because FTB doesn't have this kind of problem in NEI. And of course the carpenters blocks + EU coloured blocks crash thing. I'm not moving to 1.7.10 yet because alot of my essential mods aren't even jumping to 1.7. I'm using Japanese mods alongside with Technic ones.
